docs/addons.md

```diff
@@ -0,0 +1,240 @@
+
+Install using:
+```
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/monitoring-standalone/v1.11.0.yaml
+```
+Please note that [heapster is retired](https://github.com/kubernetes/heapster/blob/master/docs/deprecation.md). Consider using [metrics-server](https://github.com/kubernetes-incubator/metrics-server) and a third party metrics pipeline to gather Prometheus-format metrics instead.
+
+### Monitoring with Prometheus Operator + kube-prometheus
+
+The [Prometheus Operator](https://github.com/coreos/prometheus-operator/) makes the Prometheus configuration Kubernetes native and manages and operates Prometheus and Alertmanager clusters. It is a piece of the puzzle regarding full end-to-end monitoring.
+
+[kube-prometheus](https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus) combines the Prometheus Operator with a collection of manifests to help getting started with monitoring Kubernetes itself and applications running on top of it.
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/prometheus-operator/v0.26.0.yaml
+```
+
+### Route53 Mapper
+
+**This addon is deprecated. Please use [external-dns](https://github.com/kubernetes-sigs/external-dns) instead.**
+
+Please note that kOps installs a Route53 DNS controller automatically (it is required for cluster discovery).
+The functionality of the route53-mapper overlaps with the dns-controller, but some users will prefer to
+use one or the other.
+[README for the included dns-controller](https://github.com/kubernetes/kops/blob/master/dns-controller/README.md)
+
+route53-mapper automates creation and updating of entries on Route53 with `A` records pointing
+to ELB-backed `LoadBalancer` services created by Kubernetes. Install using:
+
+The project is created by wearemolecule, and maintained at
+[wearemolecule/route53-kubernetes](https://github.com/wearemolecule/route53-kubernetes).
+[Usage instructions](https://github.com/kubernetes/kops/blob/master/addons/route53-mapper/README.md)
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/route53-mapper/v1.3.0.yml
+```
+
+### Custom addons
+
+The docs about the [addon management](/development/addons.md#addon-management) describe in more detail how to define a addon resource with regards to versioning.
+Here is a minimal example of an addon manifest that would install two different addons.
+
+```yaml
+kind: Addons
+metadata:
+  name: example
+spec:
+  addons:
+  - name: foo.addons.org.io
+    version: 0.0.1
+    selector:
+      k8s-addon: foo.addons.org.io
+    manifest: foo.addons.org.io/v0.0.1.yaml
+  - name: bar.addons.org.io
+    version: 0.0.1
+    selector:
+      k8s-addon: bar.addons.org.io
+    manifest: bar.addons.org.io/v0.0.1.yaml
+```
+
+In this example the folder structure should look like this;
+
+```
+addon.yaml
+  foo.addons.org.io
+    v0.0.1.yaml
+  bar.addons.org.io
+    v0.0.1.yaml
```

Comment:
I know that this is just copy/pasted but this code block doesn't make any sense to me. Maybe someone can clarify this with a `tree` output or something in a separate PR.